SimPEG/Directives.py

```diff
@@ -732,10 +736,10 @@ def endIter(self):
 
                 if reg.eps_p > self.floorEps_p and self.coolEps_p:
                     reg.eps_p /= self.coolEpsFact
-                    print('Eps_p: ' + str(reg.eps_p))
+
                 if reg.eps_q > self.floorEps_q and self.coolEps_q:
                     reg.eps_q /= self.coolEpsFact
-                    print('Eps_q: ' + str(reg.eps_q))
+                    print(reg.eps_q)
```

Comment:
@fourndo: can you take a quick look here: do you want to also print out the name of the variable that is being displayed?